core/src/main/resources/jenkins/slaves/Messages.properties

```diff
@@ -20,8 +20,8 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-JnlpSlaveAgentProtocol.displayName=Java Web Start Agent Protocol/1 (deprecated, unencrypted)
-JnlpSlaveAgentProtocol2.displayName=Java Web Start Agent Protocol/2 (deprecated, unencrypted)
-JnlpSlaveAgentProtocol3.displayName=Java Web Start Agent Protocol/3 (deprecated, basic encryption)
-JnlpSlaveAgentProtocol4.displayName=Java Web Start Agent Protocol/4 (TLS encryption)
+JnlpSlaveAgentProtocol.displayName=Inbound TCP Agent Protocol/1 (deprecated, unencrypted)
+JnlpSlaveAgentProtocol2.displayName=Inbound TCP Agent Protocol/2 (deprecated, unencrypted)
+JnlpSlaveAgentProtocol3.displayName=Inbound TCP Agent Protocol/3 (deprecated, basic encryption)
+JnlpSlaveAgentProtocol4.displayName=Inbound TCP Agent Protocol/4 (TLS encryption)
```

Comment:
Usually seen only when you open an advanced-mode button.

Reply:
This seems to be the only place where it is referred to as "Inbound TCP" rather than just "Inbound". This difference seems a little confusing. I haven't quite figured out why the "TCP" part is important or what it needs to say. Would it be just as well to call it "Inbound Agent Protocol/x"?

Reply:
See second paragraph of https://github.com/jenkinsci/jenkins/pull/3998#issue-272398333. It is true that in this context the **TCP** could probably be dropped—_any_ of these protocols are running on the same TCP port.